README.adoc

```diff
@@ -39,22 +39,64 @@ TIP: Use https://github.com/creationix/nvm/blob/master/README.md[nvm] to manage
 :numbered:
 === Clone the project
 
-First you need to clone the https://github.com/integr8ly/tutorial-web-app-walkthroughs[walkthroughs repository] next to the tutorial web app
+First you need to clone the https://github.com/integr8ly/tutorial-web-app-walkthroughs[walkthroughs repository] next to the tutorial web app:
 
 [source,shell]
 ----
-$ git clone https://github.com/integr8ly/tutorial-web-app-walkthroughs.git
+home:$ git clone https://github.com/integr8ly/tutorial-web-app-walkthroughs.git
+home:$ cd tutorial-web-app-walkthroughs
+home:~/tutorial-web-app-walkthroughs$ install yarn
+home:~/tutorial-web-app-walkthroughs$ cd ..
 ----
 
 === Install dependencies
-Then change to the webapp directory, install its dependencies and run the development server:
+Then change to the webapp directory, install its dependencies and run the development server.  
+[source,shell]
+----
+home:$ cd tutorial-web-app
+home:~/tutorial-web-app$ yarn install
+home:~/tutorial-web-app$ yarn start:dev
+----
+
+=== Cluster configuration
+If you are running tutorial-web-app against your cluster, then you will need to configure your cluster to allow callback to your localhost:
+
+
 
 [source,shell]
 ----
-$ yarn install
-$ yarn start:dev
+$ ssh -i ~/.ssh/ocp-workshop.pem user@bastion.example.openshiftworkshop.com
```

Comment:
I don't think we should have any reference to hosts here. A user doesn't necessarily need to ssh onto the host, they could also login with the `oc` tool. maybe something like:

```
oc login <my-cluster>
oc edit oauthclient tutorial-web-app
```

and removing the `exit` further down the file would be more generic

Reply:
Yes, that would be better.